app/views/companies/show.html.haml

```diff
@@ -1,51 +1,49 @@
-%header.entry-header
-  %h1.entry-title= @company.name
-  .post-title-divider
+%div{ :id => "#{@company.class.name.downcase}-#{@company.id}", class: "show-item #{@company.class.name.downcase} #{@company.class.name.downcase}-#{@company.id}" }
```

Comment:
It's more of a convention:  CSS names and ids are lowercase.  

I'm not sure if there's any other reason for it, so although I lean towards downcasing, we can do wither.